SUPPORT.md

```diff
@@ -1,7 +1,7 @@
 # How to get support from the Libero Community
 The best place to start for support on any aspect of Libero is the [Libero Commmunity Slack workspace](https://libero.pub/join-slack/). Invitiations are open so just join the conversation and ask your question in the relevant channel.
 
-The community has members from a number of organisations and the project lead, eLife, have their [teams listed in the Libero GitHub organisation](https://github.com/orgs/libero/teams/elife-sciences/teams). If you want to mention someone on Slack, in an issue or in a pull request, then you can find out who we are by using [the teams page](https://github.com/orgs/libero/teams/elife-sciences/teams).
+The community has members from a number of organisations and the project lead, eLife, have their [people listed in the Libero GitHub organisation](https://github.com/orgs/libero/people). If you want to mention someone on Slack, in an issue or in a pull request, then you can find out who we are by using [the people page](https://github.com/orgs/libero/people).
```

Comment:
due to organization visibility setting on the single users, https://github.com/orgs/libero/people shows 7 people to anonymous users and 32 when you are in the organization

Reply:
This is perhaps a positive thing as it means we can give people the option to make themselves visible to people following links in `support.md` but does mean we have a bit of admin to do.